test/versions/failing-versions.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  'qunitjs': ['1.9.0', '1.10.0', '1.11.0', '1.12.0-pre', '1.12.0', '1.13.0',
```

Comment:
Please add comments here about known issues, and why we're ignoring them. Can be very brief, but better now while we still know what's going on.